cfme/tests/cloud_infra_common/test_vm_instance_analysis.py

```diff
@@ -338,16 +328,15 @@ def ssa_analysis_profile(appliance):
         collected_files.append({"Name": file, "Collect Contents?": True})
 
     analysis_profile_name = 'default'
-    analysis_profiles_collection = appliance.collections.analysis_profiles
-    analysis_profile = analysis_profiles_collection(
+    analysis_profile_collection = appliance.collections.analysis_profiles
+    analysis_profile = analysis_profile_collection.create(
```

Comment:
This won't work correctly, the next lines will delete it.

We need an `instantiate` call in order to check if it exists and delete, then a 2nd call to `create` to create it with the required arguments and get back an instance we can yield.

Reply:
Okay updating it.